src/naemon/checks_service.c

```diff
@@ -242,6 +242,9 @@ static int run_scheduled_service_check(service *svc, int check_options, double l
 	int macro_options = STRIP_ILLEGAL_MACRO_CHARS | ESCAPE_MACRO_CHARS;
 	int neb_result = OK;
 
+	/* get the command start time */
+	gettimeofday(&start_time, NULL);
```

Comment:
Do we need this? It is reset to `0L` a few lines down.

As a result of moving this here, before the initialization below, do we not always set that check result start & endtimes to 0 effectively ? (L313)